### PR TITLE
Migrated Behat tests from Travis CI to Github Actions

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Migrated Behat tests from Travis CI to Github Actions
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22578)

### CODE REVIEW

**Review by PierreRambaud — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @Progi1984

File: .github/workflows/behat.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+            -   name: PrestaShop Configuration
+                run: cp .github/workflows/phpunit/parameters.yml app/config/parameters.yml
+
+            -   name: Get Composer Cache Directory
+                id: composer-cache
+                run: |
+                    echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+            -   name: Cache Composer Directory
```
Comment (mvorisek) [inline .github/workflows/behat.yml:38]: the names are swapped :)

Comment (PululuK) [reply .github/workflows/behat.yml:38]: Nice catch @mvorisek 
Can you provide a PR ? I can do it if you want !

Comment (mvorisek) [reply .github/workflows/behat.yml:38]: feel free to do so, I do not need any credit for it

Comment (PululuK) [reply .github/workflows/behat.yml:38]: Done ! Thanks @mvorisek

**Pull request merged**